src/app/courses-page/load-more/load-more.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-load-more',
+  templateUrl: './load-more.component.html',
+  styleUrls: ['./load-more.component.scss']
+})
+export class LoadMoreComponent implements OnInit {
```

Comment:
again, it's a simple button. Not worth creating a separate component

Reply:
@SteveRusin button was moved to courses-list template